ani-cli

```diff
@@ -231,8 +228,8 @@ update_history() {
 download() {
     case $1 in
         *m3u8*)
-            if command -v "hls"; then
-                hls -ro "$download_dir/$2" "$1"
+            if command -v "yt-dlp" >/dev/null; then
+                yt-dlp "$1" --no-skip-unavailable-fragments --fragment-retries infinite -N 16 -o "$download_dir/$2.mp4"
```

Comment:
`yt-dlp "$1" --no-skip-unavailable-fragments --fragment-retries infinite -N 16 --paths "$download_dir" --paths temp: ${HOME}/.cache/yt-dlp-temp --output "$2.mp4"`

Reply:
> `yt-dlp "$1" --no-skip-unavailable-fragments --fragment-retries infinite -N 16 --paths "$download_dir" --paths temp: "${HOME}/.cache/yt-dlp-temp" --output "$2.mp4"`

this is for windows right?, also you can install yt-dlp from scoop iirc

Reply:
what I mean is the yt-dlp temp path isn't it a matter of how to install it?

Reply:
If we explicitly set the temp path I'd be for `mktemp -d`